fbw-a380x/docs/a380-simvars.md

```diff
@@ -307,6 +307,11 @@
 - A32NX_FMS_PAX_NUMBER
     - Number
     - Number of passengers entered on FMS/ACTIVE/FUEL&LOAD page
+## Flight Management System ATA 22
```

Comment:
```suggestion

## Flight Management System ATA 22

```
Need a blank line before and after headings for valid Markdown. I suggest installing https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint.

Reply:
fixed